vmdb/app/presenters/menu/item.rb

```diff
@@ -1,9 +1,13 @@
 module Menu
-  Item = Struct.new(:id, :name, :feature, :rbac_feature, :href, :type) do
-    def initialize(an_id, name, features, rbac_feature, href, type = :default)
+  Item = Struct.new(:id, :_name, :feature, :rbac_feature, :href, :type) do
+    def initialize(an_id, _name, features, rbac_feature, href, type = :default)
       super
     end
 
+    def name
+      _name.kind_of?(Proc) ? _name.call : _name
```

Comment:
If we make the above changes, then we can change this method to just:

```ruby
_name.call
```

Which will reduce branches at runtime.